### ISSUE TITLE
[Request]: Netflix Service

### ISSUE THREAD

**Issue by SebastianAviles** [anidl/multi-downloader-nx#525]
### Type

Both

### Suggestion

You can add Netflix service download?

**Comment by AnimeDL** [comment]
Not likely anytime soon. Netflix implements widevine and this tool doesn't yet allow for decryption of any widevine content

**Comment by SebastianAviles** [comment]
What about prime video?

**Comment by AnimeDL** [comment]
> Not likely anytime soon. Netflix implements widevine and this tool doesn't yet allow for decryption of any widevine content

This tool now supports widevine so this is on the radar again. A few things still need to be refactored to allow downloading from other services easier, but soon™

**Comment by StrPlayer** [comment]
Hi any update on this would be great having access to download from more sites

**Comment by Shiroe93** [comment]
Any news on this?

**Comment by stratumadev** [comment]
No, AniDL doesn't support MSL at the moment and will not support it in the near future.
